include/wx/aui/auibar.h

```diff
@@ -627,6 +627,12 @@ class WXDLLIMPEXP_AUI wxAuiToolBar : public wxControl
 protected:
     void Init();
 
+    // Override to return wxDefaultSize because under wxMSW wxControl::DoGetBestSize()
+    // returns DEFAULT_ITEM_HEIGHT which is not suitable as the default height/width for
+    // a horizontal/vertical toolbar. Notice that the toolbar will be resized correctly
+    // when Realize() is called.
+    virtual wxSize DoGetBestSize() const override { return wxDefaultSize; }
```

Comment:
It seems very suspicious to return `wxDefaultSize` from here. I'm not sure it breaks something, but I'd be surprised if it didn't.

Couldn't we return `m_sizer->GetMinSize()` here instead? If not, let's at least return `wxSize(1,1)` or something like this, i.e. a valid size, even if a wrong one.

Reply:
Sorry, but returning current size (i.e. `GetClientSize()`) as best size is definitely not a good idea because this means the window can never become smaller than its current size. If `GetMinSize()` has a valid value, let's return it instead. If not, returning `wxSize(1, 1)` at least shouldn't break anything (although could result in the window being too small, of course).